carbonmark-api/src/.generated/types/marketplace.types.ts

```diff
@@ -1214,6 +1223,11 @@ export type SubscriptionUsersArgs = {
   where: InputMaybe<User_Filter>;
 };
 
+export enum TokenStandard {
+  ERC20 = 'ERC20',
+  ERC1155 = 'ERC1155'
+}
+
```

Comment:
Is it used anywhere?

Reply:
not yet. but I included it on the subgraph so we have a more reliable way of checking the tokenStandard in the future to handle versus by ids or registry